fog/api/proto/view.proto

```diff
@@ -274,11 +280,31 @@ message TxOutSearchResult {
     /// FIXME: MC-1491 ensure this happens either in enclave or db, or wait for ORAM
     bytes ciphertext = 3;
 
-    /// The length of the payload that is encrypted in the ciphertext. Ciphertexts will always be 255 in length, but
-    /// the payload may be less than that, so any additional bytes should not be interpreted by the client. This value
-    /// tells the client which bytes to interpret.
-    /// TODO: Remove this field and replace with an optional `ciphertext_padding` field. This new field will contain
-    /// extra bytes to make up the difference between the shard's ciphertext length and the payload.
+    /// Unused padding that needs to be returned to maintain internal obliviousness.
+    bytes padding = 4;
+
+}
+
+/// Contains the same payload data as a TxOutSearchResult, but the payload is contained within a ciphertext of fixed
+// length.
+message FixedTxOutSearchResult {
```

Comment:
Ack. I prefer leaving out the `Size` because it's wordier without much added benefit IMO. But that makes sense for cases where we want to fix, i.e. correct, some previous API.